### PR TITLE
Add animation to modal component

### PR DESCRIPTION
**Pull request by ZhangYiJiang**
Currently some (well, one) modals are animated when they open. This PR adds the corresponding close animation and allows for it to be configured using an `animate` boolean prop.

### CODE REVIEW

File: www/src/styles/utils/animations.scss [hunk 1]
```diff
@@ -34,6 +34,21 @@
   }
 }
 
+@keyframes zoomOut {
+  from {
+    opacity: 1;
+  }
+
+  50% {
```
Comment (li-kai) [inline www/src/styles/utils/animations.scss]: Is this supposed to be here? It feels like closing animation is way too fast as a result.

Comment (ZhangYiJiang) [reply www/src/styles/utils/animations.scss]: Ah, I made a mistake mirroring the zoomIn animation. Let me fix that

File: www/src/js/views/components/Modal.scss [hunk 2]
```diff
@@ -11,6 +11,21 @@
...
+  :global {
+    animation: zoomIn 0.2s;
+  }
+}
+
+:global(.ReactModal__Overlay--before-close) {
+  .animated {
+    :global {
+      // Update closeTimeoutMS prop in Modal.jsx if this duration is changed
+      animation: zoomOut 0.15s forwards;
```
Comment (li-kai) [inline www/src/js/views/components/Modal.scss]: You should indeed change Modal.jsx 😆

Comment (ZhangYiJiang) [reply www/src/js/views/components/Modal.scss]: Ooof.

**Pull request merged**